tests/virtualization/universal/download_image.pm

```diff
@@ -0,0 +1,34 @@
+#Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Download disk image
+# Maintainer: Pavel Dostal <pdostal@suse.cz>
+
+use base "consoletest";
+use virt_autotest::common;
+use virt_autotest::utils;
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+
+    ensure_default_net_is_active();
+
+    assert_script_run "if [ ! -d \"/mnt/virt_images\" ]; then mkdir /mnt/virt_images; fi";
```

Comment:
We could also just do a `mkdir -p /mnt/virt_images` which also returns 0 if the directory already exists